packages/components/src/Spinner/Spinner.css

```diff
@@ -0,0 +1,86 @@
+.spinner {
+  position: relative;
+  pointer-events: none;
+  /* 1 */
+  opacity: 0.6;
+}
+
+.spinner:after {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  z-index: 3;
+  width: 2.25rem;
+  height: 2.25rem;
+  -webkit-box-shadow: 0px 0px 0.125rem 0.1875rem rgba(1, 27, 37, 0.3);
+  box-shadow: 0px 0px 0.125rem 0.1875rem rgba(1, 27, 37, 0.3);
+  margin: -1.125rem 0 0 -1.125rem;
+  border: 0.375rem solid rgba(1, 27, 37, 1);
+  border-top-color: rgba(255, 255, 255, 1);
+  border-radius: 50%;
+  background-color: rgba(1, 27, 37, 0.3);
+  -webkit-animation-duration: 1s;
+  animation-duration: 1s;
+  -webkit-animation-iteration-count: infinite;
+  animation-iteration-count: infinite;
+  -webkit-animation-name: animation-spin;
+  animation-name: animation-spin;
+  -webkit-animation-timing-function: linear;
+  animation-timing-function: linear;
+  will-change: transform;
+}
+
+@-webkit-keyframes animation-spin {
+  0% {
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(359deg);
+    transform: rotate(359deg);
+  }
+}
+
+@keyframes animation-spin {
+  0% {
+    -webkit-transform: rotate(0deg);
+    transform: rotate(0deg);
+  }
+  100% {
+    -webkit-transform: rotate(359deg);
+    transform: rotate(359deg);
+  }
+}
+
+/* -------------------------------------------------------------------------
+   Small Spinner
+   ------------------------------------------------------------------------- */
+.small:after {
+  top: 50%;
+  left: 50%;
+  width: 1rem;
+  height: 1rem;
+  -webkit-box-shadow: 0px 0px 1px 2px rgba(1, 27, 37, 0.4);
+  box-shadow: 0px 0px 1px 2px rgba(1, 27, 37, 0.4);
```

Comment:
Can probably use our shadow presets as well.

https://github.com/GetJobber/atlantis/tree/master/packages/design/src

Reply:
With presets:
![Screen Shot 2019-12-17 at 8 10 03 AM](https://user-images.githubusercontent.com/34727471/71007740-bdd80e00-20a4-11ea-9e2f-25ce8cf19fcf.png)

Custom:
![Screen Shot 2019-12-17 at 8 10 20 AM](https://user-images.githubusercontent.com/34727471/71007766-c7617600-20a4-11ea-8999-8dd27e991427.png)

@seanhealy I see what your saying, this css looks like garbage just hardcoded the way they are, but the preset box-shadow doesn't really look all that great on this component... Perhaps I could add to the shadow css var? Then I can call it here like:  `var(--shadow-spinner);` or something?

Reply:
We could add a `--shadow-dark` or something along those lines.

Reply:
Perfect, I put the custom css into variable `--shadow-dark` 😄